includes/class-sensei-course.php

```diff
@@ -1589,7 +1592,8 @@ public function load_user_courses_content( $user = false ) {
 			}
 
 			foreach ( $completed_courses as $course_item ) {
-				$course = $course_item;
+				$course       = $course_item;
+				$lesson_count = intval( Sensei()->course->course_lesson_count( absint( $course_item->ID ) ) );
```

Comment:
I don't think we need the `intval` here or in any of the following changed lines of code. `course_lesson_count` returns the return value of the `count` function, which is always an `int`.